setup/steps/install_python.sh

```diff
@@ -8,9 +8,15 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $DIR/../_common.sh
 set -euo pipefail
 
-MAMBAFORGE_VERSION="4.14.0-1"
-MAMBAFORGE_INSTALLER="Mambaforge-$MAMBAFORGE_VERSION-Linux-$(uname -p).sh"
-MAMBAFORGE_URL="https://github.com/conda-forge/miniforge/releases/download/$MAMBAFORGE_VERSION/$MAMBAFORGE_INSTALLER"
+MAMBAFORGE_VERSION="$@"
+[ -z "$MAMBAFORGE_VERSION" ] && MAMBAFORGE_VERSION="4.14.0-1"
```

Comment:
> I'm getting us upgraded to a newer version of Mambaforge across the board so this will hit a rebase / merge conflict once [37046a1](https://github.com/magao-x/MagAOX/commit/37046a195b9b457003a45171ce70d7296834623b) and [ccd186d](https://github.com/magao-x/MagAOX/commit/ccd186de419ebdeaf22049090df38b18e0173172) make it into the resurrector branch. Just realized the latter hadn't been pushed yet, sorry about that.

We'll cross that bridge when we come to it.

This will affect no provisioning script that does not pass the Mambaforge version as an argument to this script.